src/app/view/whiteButton.js

```diff
@@ -0,0 +1,7 @@
+export default function ButtonWhite(text) {
+  const ButtonWhite = document.createElement('button');
+  newButtonWhite.classList.add('game__btn');
+  newButtonWhite.innerText = text;
```

Comment:
The buttons have the icons in it, perhaps you could add them also? You might need to change the icon depending on the current view, maybe we can think of a way to do it without the need to rerender the whole button, just by changing the text.